### PR TITLE
scripts/set-alias-page.py, scripts/set-more-info-link.py: sync documentation and get_tldr_root

### PR DESCRIPTION
**Pull request by sebastiaanspeck**

### CODE REVIEW

File: scripts/set-alias-page.py [hunk 1]
```diff
@@ -8,7 +8,9 @@
 isn't suggested to use the sync option. If used, only stage changes
 and commit verified changes for your language.
 
-Note: If there is a symlink error when using the stage flag remove the `pages.en`
+Note: Before running this script from another directory as tldr, ensure that TLDR_ROOT is set to the location
```
Comment (vitorhcl) [inline scripts/set-alias-page.py:11]:
```suggestion
Note: Before running this script, ensure that you are in the target tldr root or in one of its subdirectories. If you aren't, ensure that TLDR_ROOT is set to the location
```

File: scripts/set-more-info-link.py [hunk 2]
```diff
@@ -5,22 +5,22 @@
 This script sets the "More information" link for all translations of a page.
 It can be used to add or update the links in translations.
 
-Note: Before running this script, ensure that TLDR_ROOT is set to the location
+Note: Before running this script from another directory as tldr, ensure that TLDR_ROOT is set to the location
```
Comment (vitorhcl) [inline scripts/set-more-info-link.py:8]:
```suggestion
Note: Before running this script, ensure that you are in the target tldr root or in one of its subdirectories. If you aren't, ensure that TLDR_ROOT is set to the location
```

**Review by vitorhcl — COMMENTED** [review]
The current explanation of the execution directory is a bit confusing to me.

I have a suggestion, feel free to reword it.

**Edit.**: IMO the suggestion in my later revision is much clearer.

File: scripts/set-alias-page.py (hunk 1 above)
Comment (vitorhcl) [inline scripts/set-alias-page.py:11]:
```suggestion
Note: If the current directory or one of its parents is called "tldr", the script will assume it is the tldr root, i.e., the directory that contains a clone of https://github.com/tldr-pages/tldr. If you aren't, the script will use TLDR_ROOT as the tldr
```

File: scripts/set-alias-page.py [hunk 3]
```diff
@@ -8,7 +8,9 @@
...
+of a clone of https://github.com/tldr-pages/tldr, and 'git' is available.
```
Comment (vitorhcl) [inline scripts/set-alias-page.py:12]:
```suggestion
root. Also, ensure 'git' is available.
```

File: scripts/set-more-info-link.py (hunk 2 above)
Comment (vitorhcl) [inline scripts/set-more-info-link.py:8]:
```suggestion
Note: If the current directory or one of its parents is named "tldr", the script will assume it is the tldr root, i.e., the directory that contains a clone of https://github.com/tldr-pages/tldr. If you aren't, the script will use TLDR_ROOT as the tldr
```

File: scripts/set-more-info-link.py [hunk 4]
```diff
@@ -5,22 +5,22 @@
...
 of a clone of https://github.com/tldr-pages/tldr, and 'git' is available.
```
Comment (vitorhcl) [inline scripts/set-more-info-link.py:9]:
```suggestion
root. Also, ensure 'git' is available.
```

**Review by vitorhcl — COMMENTED** [review]
Actually, in my opinion, it's much clearer to explain the behavior of the script instead of trying to list some guidelines ("the script does this or that" vs. "ensure something is in this way").

If you agree with me, a rewording would be welcome if you have a better idea on the subdirectory part.

File: scripts/set-alias-page.py [hunk 5]
```diff
@@ -51,16 +61,14 @@ def get_tldr_root():
...
-            "\x1b[31mPlease set TLDR_ROOT to the location of a clone of https://github.com/tldr-pages/tldr.\x1b[0m"
-        )
+    f = Path(__file__).resolve()
+    if "tldr" in str(f):
+        return next(path for path in f.parents if path.name == "tldr")
+    elif "TLDR_ROOT" in os.environ:
+        return Path(os.environ["TLDR_ROOT"])
+    raise SystemExit(
+        "\x1b[31mPlease set TLDR_ROOT to the location of a clone of https://github.com/tldr-pages/tldr."
+    )
```
[the remark this answers is not in the document]
Comment (vitorhcl) [reply scripts/set-alias-page.py]: I updated your comment to add the missing parameters to the generator expression as you did in the other PR (sorry I forgot to add them in my initial suggestion :P)

**Review by vitorhcl — APPROVED** [review]
What about changing `set-page-title-link.py` too?

**Review by kbdharun — APPROVED** [review]

**Pull request merged**